zcash_primitives/src/jubjub/fs.rs

```diff
@@ -75,9 +75,9 @@ pub struct FsRepr(pub [u64; 4]);
 
 impl ::std::fmt::Display for FsRepr {
     fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
-        r#try!(write!(f, "0x"));
+        r#write!(f, "0x")?;
```

Comment:
Remove the now-unnecessary raw marker:
```suggestion
        write!(f, "0x")?;
```